metis-indexing/metis-indexing-service/src/main/java/eu/europeana/indexing/service/dao/FullBeanDao.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2007-2013 The Europeana Foundation
+ *
+ *  Licenced under the EUPL, Version 1.1 (the "Licence") and subsequent versions as approved
+ *  by the European Commission;
+ *  You may not use this work except in compliance with the Licence.
+ *
+ *  You may obtain a copy of the Licence at:
+ *  http://joinup.ec.europa.eu/software/page/eupl
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under
+ *  the Licence is distributed on an "AS IS" basis, without warranties or conditions of
+ *  any kind, either express or implied.
+ *  See the Licence for the specific language governing permissions and limitations under
+ *  the Licence.
+ */
+package eu.europeana.indexing.service.dao;
+
+import eu.europeana.corelib.solr.bean.impl.FullBeanImpl;
+import eu.europeana.indexing.service.mongo.MorphiaDatastoreProvider;
+import java.util.List;
+import org.mongodb.morphia.Datastore;
+import org.mongodb.morphia.query.Query;
+import org.mongodb.morphia.query.UpdateOperations;
+
+public class FullBeanDao {
+    private final MorphiaDatastoreProvider morphiaDatastoreProvider;
+    private final Datastore ds;
+
+    public FullBeanDao(MorphiaDatastoreProvider morphiaDatastoreProvider) {
+    	this.morphiaDatastoreProvider = morphiaDatastoreProvider;
+    	ds = getDS();
+    }
+
+    public  <T> T searchByAbout(Class<T> clazz, String about) {
+
+		return ds.find(clazz).filter("about", about).get();

```

Comment:
Create a constant for the "about" string. It is used for the same purpose in multiple locations.

Reply:
Fixed.